zkfacade/src/main/java/com/yahoo/vespa/zookeeper/ZooKeeperServer.java

```diff
@@ -113,7 +114,7 @@ private void shutdown() {
     public void run() {
         System.setProperty(ZOOKEEPER_JMX_LOG4J_DISABLE, "true");
         String[] args = new String[]{getDefaults().underVespaHome(zookeeperServerConfig.zooKeeperConfigFile())};
-        log.log(LogLevel.DEBUG, "Starting ZooKeeper server with config: " + args[0]);
+        log.log(LogLevel.DEBUG, "Starting ZooKeeper server with config file " + args[0]);
```

Comment:
Consider adding this message to the INFO message that anyway is printed

Reply:
Thanks, will do that in a later PR